dali/base/dasess.cpp

```diff
@@ -565,7 +565,8 @@ class CSessionRequestServer: public Thread
                         INode *na = queryNullNode();
                         Owned<IGroup> dummyCoven = createIGroup(1, &na);
                         dummyCoven->serialize(mb);
-                        Owned<IException> e = makeStringException(666, "Access denied!");
+                        const char *roleName = queryRoleName((DaliClientRole)role);
+                        Owned<IException> e = makeStringExceptionV(666, "Access denied! [role=%s]", roleName);
```

Comment:
Is 666 the right number?

Reply:
There aren't sensible general server client exception in this area (SDS has it's own).
Most places set error code to -1.
666 is misleading (and devilish) - I'll change to -1 for a bit of consistency.